app/lib/features/news/pages/add_news.dart

```diff
@@ -0,0 +1,301 @@
+import 'dart:io';
+
+import 'package:acter/common/providers/chat_providers.dart';
+import 'package:acter/common/widgets/acter_video_player.dart';
+import 'package:acter/features/chat/widgets/room_avatar.dart';
+import 'package:acter/features/home/widgets/space_chip.dart';
+import 'package:acter/features/news/model/news_slide_model.dart';
+import 'package:acter/features/news/news_utils/news_utils.dart';
+import 'package:acter/features/news/providers/news_post_editor_providers.dart';
+import 'package:acter/features/news/widgets/news_post_editor/select_action_item.dart';
+import 'package:acter/features/news/widgets/news_post_editor/news_slide_options.dart';
+import 'package:atlas_icons/atlas_icons.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_easyloading/flutter_easyloading.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:go_router/go_router.dart';
+
+class AddNews extends ConsumerStatefulWidget {
```

Comment:
Just minor suggestion, to keep terminologies consistent and understandable, add `Page` word with route widget.